src/lib/components/dashboard/workspace-card-list/WorkspaceCardList.svelte

```diff
@@ -97,8 +97,9 @@
           type="other"
           buttonClassProp={`rounded sparrow-fs-16 col-lg-5 col-md-10 flex-grow-1 py-0 mb-4 add-new-workspace`}
           onClick={handleCreateWorkspace}
-          buttonStyleProp={workspaces.length == 0 ? "min-height:18vh" : ""}
+          buttonStyleProp={workspaces.length == 0 ? "min-height:132px" : ""}
```

Comment:
can we use calc here

Reply:
kept it for now and all comment to check later . in my todo list already